suave/devenv/blockscout/services/docker-compose-backend.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.8'
+
+services:
+  blockscout-backend:
+    image: blockscout/blockscout-suave:${DOCKER_TAG:-latest}
```

Comment:
**suggestion:**

it would be great if we could pin the tags to immutable values.  having "latest" makes it hard to reproduce issues as `latest` tag is a constantly moving target.

Reply:
Sure, implicit versions of Docker images configured here https://github.com/flashbots/suave-geth/pull/45/commits/590e6657b975bf74f84832c314a30d27049da53f